lib/deaf_grandma.rb

```diff
@@ -14,13 +14,27 @@ def run!
     loop do
       user_input = get_user_input
       p speak(user_input)
+      if @bye_counter > 0
+        exit(0)
+      end
     end
   end
 
 
   def speak(input)
 
-    #Implement your code here <<<<<<<<<
+    response = "SPEAK UP SONNY!"
+
+    if input.upcase!.nil?
+      response = "NOT SINCE 1964!"
+    end
+
+    if input.upcase == "BYE"
```

Comment:
@chivygab The spec says you must yell "BYE" to get the response. Here, you take any case of 'bye'.